lib/posix/options/Kconfig.compat

```diff
@@ -14,10 +17,16 @@ menuconfig EVENTFD
 	  be used as an event wait/notify mechanism together with POSIX calls
 	  like read, write and poll.
 
+if EVENTFD
```

Comment:
Does not harm and can be certainly left here, but as the `EVENTFD_MAX` has `depends on EVENTFD` this `if EVENTFD` is not needed.

Reply:
Fixed!